packages/documentation-site/patternfly-docs/content/design-guidelines/styles/icons/icons.css

```diff
@@ -71,7 +71,7 @@
   margin-top: 24px;
 }
 
-.ws-icons-gridtext .ws-icon-sizes .ws-p {
+.ws-icons-gridtext .ws-icon-sizes .pf-v6-c-content--p {
   margin-bottom: 8px;
```

Comment:
This should come with default paragraph margin so I wonder if we can remove this style and live with the spacing we get for free from `.pf-v6-c-content--p`